src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java

```diff
@@ -55,7 +55,7 @@ public void actionPerformed(final ActionEvent e) {
      * @return a JButton.
```

Comment:
Is miss a space empty between line 54 and 55

then the description of all returns should be more descriptive

Reply:
I think is missing a space empty between line 46 and 47 too in that class